openvdb/cmd/openvdb_print.cc

```diff
@@ -141,7 +245,18 @@ printLongListing(const StringVec& filenames)
                 grid->print(std::cout, /*verboseLevel=*/11);
                 firstGrid = false;
             }
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+            // Check for grid type, inspect points if this is a point grid
+            const openvdb::GridBase::ConstPtr grid = *it;
+            Name gridType = grid->type();
+            if (gridType.find("ptdata") != std::string::npos)
+                PointStats::inspectPoints(grid, pointStats);
+        }
+        // Print out point stats only if there are any points
+        if (pointStats.total != 0) PointStats::printPointStats(pointStats);
```

Comment:
We would still want to print some information to show the grid is empty, as it may contain topology but no points